cs-mail/src/main/java/io/cloudslang/content/mail/actions/GetMailAttachmentAction.java

```diff
@@ -82,6 +82,8 @@
      *                                   in destination.
      *                                   Valid values: true, false.
      *                                   Default value: false.
+     * @param deleteUponRetrieval        If true the email from which the attachment is retrieved will be deleted.
+     *                                   For any other values the email will not be deleted.
```

Comment:
Any other it's ambigous you can user valid and default values.

Reply:
added following line: "Valid values: true, false. Default value: false.". The behavior is the same as it is for GetMailMessage: any value is accepted actually